launcher/launcher.h

```diff
@@ -52,8 +52,10 @@ struct Launcher
         }
 
         void restart(std::string const & node) {
```

Comment:
well then I would of had to make the var once public and checked in the service, or make an accessor. this seemed simpler, but if it's bad style I can change it... What do you suggest?